app/src/pages/BookmarksPage.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import ContentTile from "@/components/content/ContentTile.vue";
+import { appLanguageIdAsRef, userPreferencesAsRef } from "@/globalConfig";
+import {
+    db,
+    DocType,
+    useDexieLiveQueryWithDeps,
+    type ContentDto,
+    type Uuid,
+} from "luminary-shared";
+import { computed } from "vue";
+import { BookmarkIcon } from "@heroicons/vue/24/solid";
+
+// Extract bookmark keys
+const bookmarkKeys = computed(() => Object.keys(userPreferencesAsRef.value.bookmarks));
+
+const bookmarkContent = useDexieLiveQueryWithDeps(
+    appLanguageIdAsRef,
+    (appLanguageId: Uuid) =>
+        db.docs
+            .where("parentId")
+            .anyOf(bookmarkKeys.value)
+            .filter((c) => {
+                const content = c as ContentDto;
+                if (content.type !== DocType.Content) return false;
```

Comment:
Documents with a `parentId` are always of type `ContentDto`, so this line is redundant